starboard/build/toolchain/BUILD.gn

```diff
@@ -14,7 +14,21 @@
 
 import("//build/config/clang/clang.gni")
 import("//build/toolchain/gcc_toolchain.gni")
+import("//starboard/raspi/shared/toolchain/raspi_shared_toolchain.gni")
 
-clang_toolchain("clang") {
-  clang_base_path = clang_base_path
+# clang_toolchain("clang") {
+#   clang_base_path = clang_base_path
+# }
+
+gcc_toolchain("clang") {
+  prefix = rebase_path("$clang_base_path/bin", root_build_dir)
+  cc = "$prefix/clang"
+  cxx = "$prefix/clang++"
+  ld = gcc_toolchain_cxx
```

Comment:
Looking for a clean way to configure this

Reply:
We can try to  use the same pattern as the one we use for setting the path:

clang_toolchain("x64") {
  clang_base_path = clang_base_path
  ld = custom_ld
}

Reply:
https://github.com/youtube/cobalt/pull/898/files#r1291615262